src/coreclr/src/jit/morph.cpp

```diff
@@ -16642,6 +16635,25 @@ void Compiler::fgMorph()
 
     EndPhase(PHASE_CLONE_FINALLY);
 
+    /* Compute bbNum, bbRefs and bbPreds */
+
+    JITDUMP("\nRenumbering the basic blocks for fgComputePreds\n");
+    fgRenumberBlocks();
+
+    noway_assert(!fgComputePredsDone); // This is the first time full (not cheap) preds will be computed.
+    fgComputePreds();
+
+    // Run an early flow graph simplification pass
+    if (opts.OptimizationEnabled())
+    {
```

Comment:
I realize this was just cut-and-pasted, but it would be nice to update the comment style.

Reply:
Sure.

FWIW there are a lot of old-style comments throughout `fgMorph`. And I'm tempted to "inline" `fgMorph` into `compCompile` so that most of the phases in the jit are captured in one method. Maybe I'll do this bigger change as a follow-up.

Reply:
That sounds good.